src/Themes/Theme.php

```diff
@@ -0,0 +1,16 @@
+<?php
+
+namespace Carnival\Themes;
+
+interface Theme {
```

Comment:
I forget if we discussed it on the call, but what do you think about a `getVersion` method?
Most themes will come in a plugin so their version number will likely be the version of the plugin, but at least the official theme won't come in a plugin so we can't rely on that relationship.

Reply:
Sure, I'll add a getVersion() method